### ISSUE TITLE
cant select model

### ISSUE THREAD

**Issue by excitedpepsi** [KradekMFC/MFCCountdown#1]
its worked well until the last couple of days.

currently if I hit the select model button, it doesn't actually select that model, it stays on the page.   

I've tried multiple computers, multiple browsers ,and another premium's computer.  Same result.

Yes, I know kradek said he's not doing further development of this since mfc implemented  countdown years ago.  
Anyone using this an have some insight to whats going on?

**Comment by KradekMFC** [comment]
I just pushed a change that fixes this issue. Basically MFC removed all the chat servers this code was configured to use. I pulled down the latest list and added it. This could happen again; I could configure it to  auto-pull, but don't have the time given this is basically a dead project. That said it should work for now.

**Comment by excitedpepsi** [comment]
thanks kradek!  
I'm amazed  that you fixed this 4 year old project on a saturday evening within 2 hours of my posting.
And to be honest, I'm disappointed in myself for not trying changing the servers.  I got a bit fixated on that google firebase link that appeared to not work.   

There is a small error in the new files.     The MFCSocket.js file.  The format of the server array doesn't match the format in the previous version of the file.   This results in an error 
"MFCSocket.js:212 WebSocket connection to 'ws://%7B0%7D.myfreecams.com:8080/fcsl' failed: Error in connection establishment: net::ERR_NAME_NOT_RESOLVED" 

I fixed this in the linked file.  (drop the .txt extension I had to use cause of github security).

[MFCSocket.js.txt](https://github.com/KradekMFC/MFCCountdown/files/1327051/MFCSocket.js.txt)
